app/themes/ukis/templates/introduction.html

```diff
@@ -12,9 +12,10 @@
 {% block business_details %}
   <div class="u-mb-m">
     {% if meta.survey.region_code == 'GB-NIR' %}
+      <img src="{{ theme_static('logo-beis.png') }}" alt="Department for Business, Energy and Industrial Strategy" width="200" />
```

Comment:
If you swap the order of the IF statement you could always have the BEIS logo and only add the NI logo inside the IF; no need for an ELSE.